src/lib.rs

```diff
@@ -493,6 +493,23 @@ impl EventListener {
         self.wait_internal(Some(deadline))
     }
 
+    /// Returns true if this listener listens to this event.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use event_listener::Event;
+    ///
+    /// let event = Event::new();
+    /// let listener = event.listen();
+    ///
+    /// assert!(listener.listens_to(&event));
+    /// ```
+    #[inline]
+    pub fn listens_to(&self, event: &Event) -> bool {
+        ptr::eq(&*self.inner, event.inner.load(Ordering::Relaxed))
```

Comment:
How important is the `Relaxed` ordering? Would it hurt performance to err on the safe side and go with `SeqCst`?

Reply:
I'm not sure, I'd be happy to make it SeqCst.

Reply:
I think it needs to be Acquire, to synchronize with setting the pointer when constructing the first listener (with enough inlining, I think this load could be moved before the store in in `Event::listen` by LLVM if its ordering is Relaxed).